hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/meta/bni/Dispatch.java

```diff
@@ -254,7 +254,7 @@ ResponseCodeEnum transferWithReceiverSigCheck(
      * @param amount                 the amount to charge
      * @param itemizeStoragePayments whether to itemize storage payments in the record
      */
-    ResponseCodeEnum chargeStorageRent(long contractNumber, long amount, boolean itemizeStoragePayments);
+    void chargeStorageRent(long contractNumber, long amount, boolean itemizeStoragePayments);
```

Comment:
why did this change?

Reply:
Given the `HandleException`-based idiom already used by the `BaseProxyWorldUpdater.commit()` steps, throwing on an error here seemed more natural.